eo-maven-plugin/pom.xml

```diff
@@ -315,6 +315,14 @@ SOFTWARE.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <configuration>
+          <source>11</source>
+          <target>11</target>
+        </configuration>
```

Comment:
@OlesiaSub remove this specifiers and try again, please

Reply:
@OlesiaSub I assume that your IDE added these lines. If so you can specify java version in your IDE like this: https://github.com/objectionary/dejump/pull/79#discussion_r1005716658